src/test/java/com/desiato/puresynth/configurations/AuthTokenFilterTest.java

```diff
@@ -0,0 +1,64 @@
+package com.desiato.puresynth.configurations;
+
+import com.desiato.puresynth.BaseTest;
+import jakarta.servlet.FilterChain;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
+
+
+import java.util.UUID;
+
+@SpringBootTest
+public class AuthTokenFilterTest extends BaseTest {
```

Comment:
It's more `AuthenticatinTest`, than jsut a filter test. It's not a unit test, but a full functional test.

Reply:
Also, isn't it duplicated with `LoginControllerTest`?